bartpy/model.py

```diff
@@ -13,7 +13,7 @@
 class Model:
 
     def __init__(self,
-                 data: Data,
+                 data: Optional[Data],
```

Comment:
You're the best kind of correct :)  Note for future self, now that individual nodes in the tree don't keep full copies of the data, the deep copy set up which makes use of a `None` parameters can probably go